packages-common/pluggable-widgets-generator/generators/app/index.js

```diff
@@ -25,19 +20,21 @@ class MxGenerator extends Generator {
                 this.widgetParamName = name;
             }
         }
+
+        this.widgetSrcFolder = "src/components/";
```

Comment:
i would say that having this as a file-level constant is more logical than as a field

Reply:
Agree, reverted it to file-level constant again.